### PR TITLE
Add Thai providers

### PR DESCRIPTION
**Pull request by tuwannu**

### CODE REVIEW

File: src/Faker/Provider/th_TH/Person.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+    );
+
+    /**
+     * @var array Thai male first names.
+     * @link https://github.com/tuwannu/thai-names/blob/master/FIRST_NAMES_MALE.txt
```
Comment (fzaninotto) [inline src/Faker/Provider/th_TH/Person.php]: Sorry, that doesn't count as a trustworthy datasource

**Comment by fzaninotto** [comment]
I suggest you remove the `Person` provider until you can reference a valid datasource.

**Comment by tuwannu** [comment]
Right. Thanks for understanding. We don't really have an official datasource or anything close at all for person's names

**Comment by tuwannu** [comment]
`Person` removed.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**